src/Tickets/Seating/Orders/Attendee.php

```diff
@@ -157,4 +159,70 @@ public function remove_move_row_action( $actions, $item ) {
 		
 		return $actions;
 	}
+	
+	/**
+	 * Include seating data into the attendee object.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post $post   The attendee post object, decorated with a set of custom properties.
+	 *
+	 * @return WP_Post
+	 */
+	public function include_seating_data( $post ) {
```

Comment:
Add check for `$post` being an actual instance of `WP_Post`. In the following code you're assuming it, but it's not type-hinted, it cannot be assumed.